modules/preview-react/text/stories/Text.stories.mdx

```diff
@@ -0,0 +1,131 @@
+import {Meta} from '@storybook/addon-docs';
+
+import {Basic as BasicText} from './examples/Text';
+import {Label} from './examples/Label';
+import {TypeLevelBody} from './examples/TypeLevelBody';
+import {TypeLevelHeading} from './examples/TypeLevelHeading';
+import {TypeLevelSubtext} from './examples/TypeLevelSubtext';
+import {TypeLevelTitle} from './examples/TypeLevelTitle';
+
+import {TextComponent, LabelComponent, TypeLevelComponent} from './PropTables.splitprops.tsx';
+
+<Meta title="Preview/Text/React" />
+
+# Canvas Kit React Text
+
+`Text` is a group of text components that provide a common, ergonomic API to render text with
+simplified usage of Canvas Kit type token.
+
+## Installation
+
+```sh
+yarn add @workday/canvas-kit-react
+```
+
+## Text
+
+### Usage
+
+`Text` is a basic text component to render modified text. `Text` component composes `Box` component
+that allows to use all layout props, such as `display`, `margin`, `lineHeight` etc. It also has
+their own props such as `level` and `size` that should be provided together, because they refer to
+type token level and size, and `variant` to set variant color.
```

Comment:
`Text` is the base component that is used to render text in the UI. It is built off of our `Box` component which allows the use of styles props such as `display`, `margin`, `padding`  as well as the most common type styles such as `fontWeight`, `fontSize` amongst others. This component is an extension of our type tokens(insert link), therefore, props like `variant`, `size` and `level` all exists.

Reply:
this is a suggestion I'm sure there's better ways to word this

Reply:
this sounds better. Thank you!